.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,10 +1,8 @@
 ---
 name: Question
-about: Ask a question from the developers
+about: Ask a question for the developers
```

Comment:
Maybe "ask the developers a question" instead?

Reply:
Sounds more concise, updated.